io.github.lucasew.regex101.yaml

```diff
@@ -0,0 +1,33 @@
+id: io.github.lucasew.regex101
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-23.08
+command: io.github.lucasew.regex101
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents/regex101.sqlite
+modules:
+  - name: build
+    sources:
+        - type: git
+          url: https://github.com/lucasew/regex101.git
+          tag: 0.3.0
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    post-install:
+      - install -Dm644 -t ${FLATPAK_DEST}/share/applications installer/regex101.desktop
```

Comment:
this should have been installed by the upstream build system.

Reply:
done